guides/15_Background_Filter_Video_Processor.md

```diff
@@ -0,0 +1,145 @@
+
+```
+Cross-Origin-Opener-Policy: same-origin
+Cross-Origin-Embedder-Policy: require-corp
+```
+
+when serving your application HTML. These headers are optional but advised: they improve web security, as well as allowing the browser to support technologies like `SharedArrayBuffer`.
+
+## Checking for support before offering background blur
+
+Some browsers support the Amazon Chime SDK but do not support background blur. Additionally, some devices are not fast enough to keep up with real-time video while blurring the background.
+
+The SDK provides a static method to allow you to cheaply check for the required browser features:
+
+```typescript
+import { BackgroundBlurVideoFrameProcessor } from 'amazon-chime-sdk-js';
+BackgroundBlurVideoFrameProcessor.isSupported();
+```
+
+If `isSupported` returns `true`, you can instantiate the blur video frame processor.
+
+```typescript
+const processors = [];
+if (BackgroundBlurVideoFrameProcessor.isSupported()) {
+    const blurProcessor = await BackgroundBlurVideoFrameProcessor.create();
+    processors.push(blurProcessor);
+}
+```
+
+The create method will pre-load the model file and prepare background blur for use. Failure to load necessary resources will result in `isSupported` returning `false`. 
+
+
+## Adding background blur to your application
+
+Background blur integrates with the SDK by providing a new implementation of the `VideoFrameProcessor`
+
+Once you have checked for support and created a blur processor, you add the processor to the transform device.
+
+```typescript
+const processors = [];
+if (BackgroundBlurVideoFrameProcessor.isSupported()) {
+    const blurProcessor = await BackgroundBlurVideoFrameProcessor.create();
+    processors.push(blurProcessor);
+}
+let transformDevice = new DefaultVideoTransformDevice(logger, device, processors);
+```
+
+## Configuration
+
+Both `isSupported` and `create` accept _specifications_ — `BackgroundFilterSpec` structures — as input. These allow you to describe the model attributes, execution approach, and other configuration options that define how the feature should operate.
+
+Most developers will not need to provide a specification: the defaults are chosen to work well and adapt to runtime constraints. 
+
+## Observer notifications
+
+You can optionally implement the `BackgroundBlurVideoFrameProcessorObserver` interface and use `addObserver` to receive callbacks when an event occurs:
+
+* `filterFrameDurationHigh`:  This event occurs when the amount of time it takes to apply the background blur is longer than expected. The measurement is taken from the time the process method starts to when it returns. For example, if the video is running at 15 frames per seconds and we are averaging more than 67 ms (1000 ms reporting period / 15 fps) to apply background blur, then a very large portion of each frame's maximum processing time is taken up by processing background blur.
+
+```typescript
```

Comment:
Should we add types to the examples if saying `typescript`? Otherwise, we can change this to `js` or `javascript` I think.

Reply:
Added the type of the observer